llvm/test/CodeGen/NVPTX/vector-stores.ll

```diff
@@ -37,3 +37,17 @@ define void @v16i8(ptr %a, ptr %b) {
   store <16 x i8> %v, ptr %b
   ret void
 }
+
+; CHECK-LABEL: .visible .func v16i8_store
+define void @v16i8_store(ptr %a, <16 x i8> %v) {
+; CHECK: st.v4.u32
```

Comment:
Can we check for the extracts as well (their absense)?

Reply:
Sure, done.